src/librustc/middle/const_eval.rs

```diff
@@ -271,6 +271,7 @@ pub enum ConstVal {
     Bool(bool),
     Struct(ast::NodeId),
     Tuple(ast::NodeId),
+    Function(ast::DefId),
```

Comment:
I was just following the Struct/Tuple definition. And it made things easier with Expr::Call